sdk/apimanagement/Azure.ResourceManager.ApiManagement/api/Azure.ResourceManager.ApiManagement.netstandard2.0.cs

```diff
@@ -4227,7 +4245,7 @@ public partial class ReportRecordContract
         public double? ServiceTimeAvg { get { throw null; } }
         public double? ServiceTimeMax { get { throw null; } }
         public double? ServiceTimeMin { get { throw null; } }
-        public string SubscriptionId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier SubscriptionId { get { throw null; } }
```

Comment:
SubscriptionId is a resource identifier? Is this correct? Or if it is, should we rename this to subscriptionResourceId?

Reply:
Yes, it's a resource identifier, see this [description](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2021-08-01/definitions.json#L4689). And I have changed its name.